### PR TITLE
Enabling other cepstral voices

### PR DESCRIPTION
**Pull request by floweisshardt**
Now there is support for David and Matthias on cob4-2

### CODE REVIEW

File: cob_mimic/package.xml [hunk 1]
```diff
@@ -20,7 +20,6 @@
 
   <exec_depend>actionlib</exec_depend>
   <exec_depend>roslib</exec_depend>
-  <exec_depend>rospy</exec_depend>
```
Comment (fmessmer) [inline cob_mimic/package.xml:23]: why is ropy deleted?

Comment (floweisshardt) [inline cob_mimic/package.xml:23]: I don't know :-)

...added again

File: cob_mimic/CMakeLists.txt [hunk 2]
```diff
@@ -15,7 +15,7 @@ add_action_files(DIRECTORY action FILES
 generate_messages(DEPENDENCIES actionlib_msgs)
 
 catkin_package(
-  CATKIN_DEPENDS actionlib_msgs message_runtime
+  CATKIN_DEPENDS actionlib_msgs message_runtime roslib
```
Comment (fmessmer) [inline cob_mimic/CMakeLists.txt]: why is `roslib` added? It's only required as a `exec_depend` in `package.xml`
Please undo and update with ipa320 indigo_dev branch

Comment (floweisshardt) [inline cob_mimic/CMakeLists.txt]: it was added by @ipa-nhg. @ipa-nhg: do you know why?

Comment (NadiaHG) [inline cob_mimic/CMakeLists.txt]: I think that it is out-of-date ( before the package xml 2 migration). This dependency was in my old branch play_sound.

Comment (floweisshardt) [inline cob_mimic/CMakeLists.txt]: deleted roslib

**Pull request merged**